config/application.rb

```diff
@@ -8,7 +8,7 @@
 Bundler.require(*Rails.groups)
 
 module Avalon
-  VERSION = '7.2.1'
+  VERSION = '7.2.2'
```

Comment:
Do you think this should be `7.3`?  I can't remember if there are any new features being added or if this is all bug fixes and dependency upgrades.